src/ResourceManager/Resources/Commands.ResourceManager/Cmdlets/Implementation/Providers/GetAzureProviderCmdlet.cs

```diff
@@ -89,5 +91,38 @@ public override void ExecuteCmdlet()
                 this.WriteObject(providers, enumerateCollection: true);
             }
         }
+
+        private PSResourceProvider[] ListPSResourceProviders()
```

Comment:
The PowerShell practice is in preference to PS abbreviation. For example, the base class PSObject, PSCustomObject. Here I think we can follow the practice.